### ISSUE TITLE
[CoE Starter Kit - QUESTION] QUESTION

### ISSUE THREAD

**Issue by gghaleb** [microsoft/coe-starter-kit#8039]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Installed the CoE but I am only gettingdata from the CoE environment, where are the data of other environments? Thanks

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

March 2024

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by Jenefer-Monroe** [comment]
Hello, Does the identity running the flow have the Power Platform Admin Role permanently and directly assigned?

**Comment by gghaleb** [comment]
Yes, but we believe the issue is in the HTTP with Microsoft Entra ID connector.

![image](https://github.com/microsoft/coe-starter-kit/assets/40804402/05a2624d-05bc-49d6-831d-3ef5625fd065)

**Comment by Jenefer-Monroe** [comment]
It is the **_HTTP with Microsoft Entra ID (preauthorized)_** connector that you should use. Can you please try with that?

**Comment by gghaleb** [comment]
I want to use it but I am not able as you can see above in the error it looks like we need the organisation to consent on specific scope..

**Comment by Jenefer-Monroe** [comment]
No we do not use the other connector. You may have added an unmanaged layer to that flow. 
Please delete the other non-preauth connector, and check for and remove unmanaged layers on that flow.

**Comment by gghaleb** [comment]
closing the issue since we reinstalled the most recent version and it worked. thanks!
